centreon/common/cisco/standard/snmp/mode/stack.pm

```diff
@@ -239,7 +239,11 @@ sub manage_selection {
 
     if ($redundant->{$oid_cswRingRedundant} != 1) {
         $self->{output}->add_option_msg(short_msg => "Stack ring is not redundant");
-        $self->{output}->option_exit();
+
```

Comment:
Just a last thing, please prefer keep option exit function call side by side.  One if is enough here : 

```
    if (($redundant->{$oid_cswRingRedundant}) != 1 && (!defined($self->{option_results}->{ignore_cswringredundant}))) {
        $self->{output}->add_option_msg(short_msg => "Stack ring is not redundant");
        $self->{output}->option_exit();
    }
```

Reply:
Yes, i understand! 

First we thought it would be helpfull to print out that the "stack ring is not redundant" here - so we're able to use the search output-filter in the Centreon GUI. I will add your recommendation. 

Further i will think about a proper solution to get a warning-message/output/status if the stack ring is not redundant (with a binary option at the command line).

Reply:
Hum yeah you're right, could be interesting. 

We could imagine to set redundancy state as a counter, this way user can set Threshold like :  

--warning-status='%{stack} =~ /Not redundant'

and see the Redundancy state in the output ? 

Are you interested in such thing ? If so I could push the relevant evolution on your fork OR even better, you could find some inspiration here (https://github.com/centreon/centreon-plugins/blob/505de51edc697f30101dfd3774c1ef06b4e4ca40/network/fortinet/fortimanager/snmp/mode/devicestatus.pm#L37) 

Let me know but think your proposition is relevant and deserve to be implemented in this PR